index.html

```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
             Languages
-              <div class="category-list-container">
-                <div class="category-list">Javascript</div>
-                <div class="category-list">Ruby</div>
-                <div class="category-list">CSS</div>
-                <div class="category-list">HTML</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Javascript</div>
+              <div class="category-list">Ruby</div>
+              <div class="category-list">CSS</div>
+              <div class="category-list">HTML</div>
+            </div>
           </li>
           <li class="frameworks">
             Frameworks
-              <div class="category-list-container">
-                <div class="category-list">React.js</div>
-                <div class="category-list">Ruby on Rails</div>
-                <div class="category-list">RSpec</div>
-                <div class="category-list">Capybara</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">React.js</div>
+              <div class="category-list">Ruby on Rails</div>
+              <div class="category-list">RSpec</div>
+              <div class="category-list">Capybara</div>
+            </div>
           </li>
           <li class="skills">
             Skills
-              <div class="category-list-container">
-                <div class="category-list">Database Management</div>
-                <div class="category-list">Version Control</div>
-                <div class="category-list">CLI</div>
-                <div class="category-list">API Design</div>
-                <div class="category-list">Web Development</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Database Management</div>
+              <div class="category-list">Version Control</div>
+              <div class="category-list">CLI</div>
+              <div class="category-list">API Design</div>
+              <div class="category-list">Web Development</div>
+            </div>
           </li>
         </ul>
       </div>
     </section>
+
+
+    <!--  contact form section -->
+    <!-- added by habtamualemayehu16@gmail.com   -->
+    <section class="reg-form">
+      <div class="form-body">
+        <div class="header-text">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        </div>
+        <form action="https://formspree.io/f/xknaqwwd" method="post">
+          <input type="text" class="text-field" placeholder="Full name" min="1" max="30" name="Name" required>
+          <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+          <textarea name="Bio" id="bio" class="text-field" cols="30" rows="5" min="1" max="500" placeholder="Write me something"></textarea required>
```

Comment:
- [ ] I noticed even though your  `textarea` has the `required` keyword,  it is not being used in the right place. As a result, the form was submitted even if it wasn't filled. Kindly fix this to ensure that your `Form` is properly validated :+1: 
- [ ] I also noticed that you didn't use the right keyword for ensuring that these requirements are met. You should use the `maxlength` attribute and not `max` as you've used currently :+1: So kindly fix these as well and ensure that the below requirements are met :+1: 
```
Name field accepts up to 30 chars.
Message field accepts up to 500 chars.
```